mappings/net/minecraft/block/CopperBulbBlock.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/unmapped/C_uodosueo net/minecraft/block/CopperBulbBlock
+	FIELD f_opgqdqwv degradationLevel Lnet/minecraft/unmapped/C_nxfuouvr$C_cencorqf;
```

Comment:
```suggestion
	FIELD f_opgqdqwv oxidizationLevel Lnet/minecraft/unmapped/C_nxfuouvr$C_cencorqf;
```
degradation? this should be `oxidization`, I'd rather name it based on the field type instead of the fact that the block implements `Degradable`

Reply:
Mmm yes I agree, I forget why I did that